src/components/button/Button.styles.js

```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+    buttonEnabled: {
+        backgroundColor: '#7199FF',
+        borderRadius: 8,
+        height: 60,
+        width: 319,
```

Comment:
Don't use fixed width

Reply:
This has been changed to use percentage dimensions